APIMatic.Core/Pagination/PaginatedResult.cs

```diff
@@ -0,0 +1,16 @@
+﻿using APIMatic.Core.Types.Sdk;
+
+namespace APIMatic.Core.Pagination
+{
+    public class PaginatedResult<TPage>
+    {
+        public CoreResponse Response { get; }
+        public TPage PageMetadata { get; }
```

Comment:
This name of the property is confusing, it represents a `Page` but named as `PageMetadata`